src/astro/sql/operators/load_table.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Callable, Optional
+
+from airflow.decorators.base import get_unique_task_id
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.databases import create_database
+
+
+class LoadTableOperator(BaseOperator):
+    """
+    Load the result of a SQL statement
```

Comment:
We should mention what it returns. WDYT?

Reply:
Rephrase it please let me know WDYT?